test/smoke/framework/util.go

```diff
@@ -0,0 +1,125 @@
+// Copyright 2019 Red Hat, Inc. and/or its affiliates
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package framework
+
+import (
+	"fmt"
+	"io/ioutil"
+	"math/rand"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/DATA-DOG/godog/gherkin"
+	"github.com/kiegroup/kogito-cloud-operator/pkg/logger"
+	"github.com/kiegroup/kogito-cloud-operator/pkg/util"
+
+	"go.uber.org/zap"
+)
+
+// GetLogger retrieves the logger for a namespace
+func GetLogger(namespace string) *zap.SugaredLogger {
+	return logger.GetLogger(fmt.Sprintf("%s - tests", namespace))
+}
+
+// GenerateNamespaceName generates a namespace name, taking configuration into account (local or not)
+func GenerateNamespaceName() string {
+	rand.Seed(time.Now().UnixNano())
+	ns := fmt.Sprintf("cucumber-%s", RandSeq(4))
+	if local := util.GetOSEnv("LOCAL_TESTS", "false"); local == "true" {
+		username := util.GetOSEnv("USERNAME", "none")
+		ns = fmt.Sprintf("%s-local-%s", username, ns)
+	}
+	return ns
+}
+
+// GenerateExamplesString generates a string of values from
+func GenerateExamplesString(examples []*gherkin.Examples) string {
```

Comment:
What is this used for?

Reply:
This is used to display the examples in a scenario outline.
was supposed to be used here: https://github.com/kiegroup/kogito-cloud-operator/pull/141/files#diff-5286f02b120ae078bcb07466f75adab1R55 ...
will update